sonatype-ci-release/src/main/scala/org/typelevel/sbt/TypelevelSonatypeCiReleasePlugin.scala

```diff
@@ -78,9 +78,17 @@ object TypelevelSonatypeCiReleasePlugin extends AutoPlugin {
                 |resolvers += "${repo.name}" at "${repo.root}"
                 |```
                 |""".stripMargin
+
+          case repo: URLRepository =>
+            s"""|```scala
+                |resolvers += Resolver.url("${repo.name}", ${repo.patterns}, ${repo.allowInsecureProtocol})
```

Comment:
Thanks, this is looking much better! But this API doesn't seem to exist 🤔 
https://github.com/sbt/librarymanagement/blob/8d71f66997c356790fc887b17e33c7be66a4e590/core/src/main/scala/sbt/librarymanagement/ResolverExtra.scala#L350-L361

Reply:
corrected to use `URLRepository.apply`